src/reducers/org.js

```diff
@@ -183,15 +184,14 @@ const updateCookiesOfParentOfHeaderWithId = (state, headerId) => {
 };
 
 const advanceTodoState = (state, action) => {
-  const headerId = action.headerId || state.get('selectedHeaderId');
-  if (!headerId) {
+  const { headerId, logIntoDrawer, currentDate } = action; // TODO: isn't logIntoDrawer a setting in the store? then why to pass it?
```

Comment:
@schoettl Because it's not in _the_ store. Not in the `org` store that is. It's in the `base` store.